### PR TITLE
Automated cherry pick of #15902

### PR DESCRIPTION
**Pull request by janetkuo**
Automated cherry pick of #15902 upstream release 1.1

### CODE REVIEW

**Comment by deads2k** [comment]
This is neat.

lgtm. @lavalamp What's the process for these?

**Comment by zmerlynn** [comment]
@deads2k: In practice the mergebot uses completely unrelated signals on head and then decides randomly to merge. In theory, once it's kermit-y here, I'm happy to merge it to `release-1.1`.

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -1384,7 +1384,7 @@ func printHorizontalPodAutoscaler(hpa *extensions.HorizontalPodAutoscaler, w io.
 		reference,
 		target,
 		current,
-		minPods,
+		*minPods,
```
Comment (lavalamp) [inline pkg/kubectl/resource_printer.go]: how do you know minPods is not nil?

Comment (lavalamp) [inline pkg/kubectl/resource_printer.go]: How did this compile previously?

Comment (lavalamp) [inline pkg/kubectl/resource_printer.go]: oh-- it's printf, I see.

**Comment by lavalamp** [comment]
@deads2k: what @zmerlynn said.

File: pkg/kubectl/resource_printer.go (hunk 1 above)
Comment (JanetKuo) [inline pkg/kubectl/resource_printer.go]: handle this in #15997

**Pull request merged**